documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -898,7 +898,11 @@ If the number of retries exceeds the retry value, the sink connector enters a _F
 |[[jdbc-property-flush-retry-delay-ms]]<<jdbc-property-flush-retry-delay-ms, `+flush.retry.delay.ms+`>>
 |1000
 |Specifies the number of milliseconds that the connector waits to retry a flush operation that failed.
-
+[NOTE]
+====
+When you set both the `flush.retry.delay.ms` and xref:jdbc-property-flush-max-retries[`flush-max-retries`] properties, it can affect the behavior of the Kafka link:https://kafka.apache.org/documentation/#consumerconfigs_max.poll.interval.ms[max.poll.interval.ms] property.
```

Comment:
```suggestion
When you set both the `flush.retry.delay.ms` and xref:jdbc-property-flush-max-retries[`flush.max.retries`] properties, it can affect the behavior of the Kafka link:https://kafka.apache.org/documentation/#consumerconfigs_max.poll.interval.ms[max.poll.interval.ms] property.
```